src/lib/bootloader/grub2base.rb

```diff
@@ -400,6 +408,18 @@ def trusted_boot_summary
           "<a href=\"enable_trusted_boot\">(" + _("enable") + ")</a>"
         end
     end
+
+    # Update nvram shown in summary screen
+    #
+    # @return [String]
+    def update_nvram_summary
```

Comment:
is this summary used anywhere?

Reply:
src/lib/bootloader/grub2.rb:      result << secure_boot_summary if Systeminfo.secure_boot_available?(name)
src/lib/bootloader/grub2.rb:      result << trusted_boot_summary if Systeminfo.trusted_boot_available?(name)
src/lib/bootloader/grub2.rb:      result << update_nvram_summary if Systeminfo.nvram_available?(name)
src/lib/bootloader/grub2.rb:      result << url_location_summary unless no_location
src/lib/bootloader/grub2.rb:      order_sum = disk_order_summary
src/lib/bootloader/grub2.rb:    def disk_order_summary
src/lib/bootloader/grub2.rb:    def url_location_summary
src/lib/bootloader/grub2base.rb:    def secure_boot_summary
src/lib/bootloader/grub2base.rb:    def trusted_boot_summary
src/lib/bootloader/grub2base.rb:    def update_nvram_summary
src/lib/bootloader/grub2efi.rb:      result << secure_boot_summary if Systeminfo.secure_boot_available?(name)
src/lib/bootloader/grub2efi.rb:      result << trusted_boot_summary if Systeminfo.trusted_boot_available?(name)
src/lib/bootloader/grub2efi.rb:      result << update_nvram_summary if Systeminfo.nvram_available?(name)